roles/vmware-ova/ovfconf/files/ovfconf

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+ADDRESS=`/usr/bin/vmtoolsd --cmd "info-get guestinfo.ovfenv" | /bin/grep 'key="address"' | /bin/sed 's/.*value="\([^"]*\)".*/\1/'`
```

Comment:
This is a very fragile way to parse XML. We should be using a language for which actual XML parsers are available, in our case, Python.